be/src/common/logconfig.cpp

```diff
@@ -25,8 +25,7 @@
 #include <mutex>
 
 #include "common/config.h"
-#include "gutil/stringprintf.h"
-#include "util/logging.h"
+#include "common/logging.h"
```

Comment:
Maybe it's redundant?

Reply:
this cpp file implement `init_glog` declared in `common/logging.h`